src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGeneratorSettings.cs

```diff
@@ -27,6 +27,10 @@ public CodeGeneratorSettings()
 
         public GenerateTarget Target { get; set; }
 
+        public string StablePackageVersion { get; set; }
```

Comment:
@JakeRadMSFT @LittleLittleCloud - what do you think of this approach? It at least removes hard-coded versions from the CodeGenerator assembly, and allows the "app" (CLI and Model Builder) to pick the version they require.